columnflow/tasks/framework/mixins.py

```diff
@@ -2479,10 +2486,13 @@ def hist_hooks_repr(self) -> str:
         """
         Return a string representation of the hist hooks.
         """
-        hooks = [hook for hook in self.hist_hooks if hook not in (None, "", law.NO_STR)]
+        if self.hist_hooks_short_repr:
+            hooks_repr = law.util.create_hash(self.hist_hooks)
```

Comment:
I'd suggest to add the number of used hist-hooks to the output path because it can help immensely to distinguish output paths of different combinations of hist hooks.
```suggestion
            hooks_repr = f"{len(self.hist_hooks))}__{law.util.create_hash(self.hist_hooks)}"
```

Reply:
sounds great, I also added independency on the order of the hooks at the same time.